src/main/java/duke/exception/ErrorString.java

```diff
@@ -0,0 +1,44 @@
+package duke.exception;
+
+/**
+ * ErrorString is an enum that contains error strings for Duke.
+ *
+ * @author yl-ang
+ */
+public enum ErrorString {
```

Comment:
Very nice way of keeping all the error messages centralized. I like how it makes things easy to change error messages.